### PR TITLE
Added error message for prompt-based loss and num_variants=1

### PR DESCRIPTION
**Pull request by Benw8888**
The loss functions ccs_prompt_var and prompt_var_squared cause NaN errors when --num_variants is set to 1, because of the variance calculation.

I added an error message in train.py for this.

As ccs_prompt_var and num_variants=1 are set by default, this means an error is thrown by default, which is not ideal.

### CODE REVIEW

**Review by norabelrose — APPROVED** [review]
LGTM, thanks

**Comment by AlexTMallen** [comment]
Ah it looks like torch.var by default computes the sample variance, which divides by n-1, rather than n. The previous code was dividing by n so returned the equivalent of CCS loss. But your solution seems fine too, since we don't really care about being able to run prompt_var with num_variants=1

**Comment by norabelrose** [comment]
@AlexTMallen yeah you can also do `unbiased=False` but I think it's better to just error probably
